packages/next/src/build/webpack-config.ts

```diff
@@ -2037,6 +2037,7 @@ export default async function getBaseWebpackConfig(
   }
 
   const configVars = JSON.stringify({
+    optimizePackageImports: config?.experimental?.optimizePackageImports,
```

Comment:
probably also add other bundling related configs like `transpilePackages` as well?

Reply:
Those are different as they don't affect compilation in this case, i.e. that affects `externals` output which would invalidate the cache.